muted-tests.yml

```diff
@@ -537,6 +537,38 @@ tests:
 - class: org.elasticsearch.xpack.esql.CsvTests
   method: test {csv-spec:ip.CdirMatchEqualsInsOrs}
   issue: https://github.com/elastic/elasticsearch/issues/132860
+  method: test {csv-spec:mv_percentile.FromIndexPercentile}
+  issue: https://github.com/elastic/elasticsearch/issues/132846
```

Comment:
seems weird, shouldn't there be another `class` line?

Reply:
Yes seems like merge conflict resolved badly, will fix. Thanks!